recipes/oq-engine/meta.yaml

```diff
@@ -0,0 +1,79 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
```

Comment:
Can you remove the comments from the recipe? (You can do that later in the feedstock.)